nvtabular/io.py

```diff
@@ -405,6 +406,12 @@ def set_col_names(self, labels, cats, conts):
         self.labels = labels
         self.column_names = labels + conts
 
+    def set_col_idx(self, gdf):
+        counter = 0
+        for x in gdf.columns.values:
+            self.col_idx[str(x)] = counter
+            counter += 1
+
```

Comment:
you should use pythons ```enumerate``` for this instead of maintaining a count yourself
```suggestion
        for i, x in enumerate(gdf.columns.values):
            self.col_idx[str(x)] = i
```